Carbonara SPA.txt

```diff
@@ -0,0 +1,13 @@
+{
+ "ticker": "CARBO",                                                       
+ "poolId": "8b8417759d9a4361d475945e1f8ff8d367a09d4afd96a509bffec9e0",  
+ "social": {                                                            
+   "twitter": "CarbonaraPool",
+   "facebook": "",  
+   "instagram": " ",
+   "youtube": " "
+ },
+ "telegram": "Yiannis (CARBO)",                                             

```

Comment:
Please create a user_name on telegram and use that.
Example: 
![image](https://user-images.githubusercontent.com/71383747/93664874-c073f380-fa7a-11ea-9696-780d4d67d722.png)

![image](https://user-images.githubusercontent.com/71383747/93664892-db466800-fa7a-11ea-944e-14b7edd407a9.png)